src/server/test/pegasus_server_write_test.cpp

```diff
@@ -67,7 +67,12 @@ class pegasus_server_write_test : public pegasus_server_test_base
                 for (int i = put_rpc_cnt; i < total_rpc_cnt; i++) {
                     writes[i] = pegasus::create_remove_request(key);
                 }
-                auto cleanup = dsn::defer([=]() { delete[] writes; });
+                auto cleanup = dsn::defer([=]() {
```

Comment:
`for (auto &write : writes)`

Reply:
`error: ‘begin’ was not declared in this scope`
It is inconvenient to use iter in pointer